contrib/cardump/cardump.c

```diff
@@ -169,8 +170,19 @@ int screenio(void) {
 
 			printf("%d\t%x\t%x\t%02x\t%d\t", (int)time(NULL),frame.src.type, frame.dst.type, frame.type, frame.len);
 			for (int i=0;i<frame.len;i++) printf("%02x ", frame.payload[i]);
+			printf("\nASCII TRANSLATION:\n");
```

Comment:
cardump outputs TSV on STDOUT and human readable output on STDERR such that
`catdump < /dev/ttyUSB0 > mydata.tsv` will output a stream in a machine readable form to `mydata.tsv` and a more human readable form to the console. So these prints should go to stderr.